readme.md

```diff
@@ -333,7 +335,7 @@ The refinement and confidence in the schedule gets better and better each iterat
 **Tip 19: Iterate the Schedule with the Code**
 
 ### What to Say When Asked for an Estimate
-**"I'll get back to you."**
```

Comment:
I will keep the "". Makes it easier to see that is a sentence to say.

Reply:
sure!